bump_stargate.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
who will call that script? Will it be called automatically or manually?

Reply:
Both possible.